docs/running-node-manual/genesis-vn.md

```diff
@@ -0,0 +1,159 @@
+# Running a Genesis Validator Node manually
+
+## Prerequisites
+Make sure you have all [prerequisites](./prerequisites.md) set up
+## Deployment steps
+
+#### 1. Put `cosmovisor` binary to `/usr/bin/`, set proper owner and execution permissions.
+
+#### 2. Locate the genesis app version to genesis application version directory:
+- Create `$HOME/.dcl/cosmovisor/genesis/bin` directory.
+- Copy `dcld` binary to it, set proper owner and execution permissions.
+    Please note that execution permissions on `dcld` should be granted to all (i.e. User, Group and Others classes)
+    because cosmovisor requires execution permission on the application binary to be granted to Others class.
+
+#### 3. Choose the chain ID. Every network (for example, test-net, main-net, etc.) must have a unique chain ID.
+
+#### 4. Configure CLI:
+- `./dcld config chain-id <chain-id>` - the chosen unique chain ID.
+- `./dcld config output json` - Output format (text/json).
+
+#### 5. Create keys for a node admin and a trustee genesis accounts
+
+```bash
+./dcld keys add "<key-name>" 2>&1 | tee "<key-name>.dclkey.data"
+```
+Expected output format:
+
+```json
```

Comment:
Do we really need to provide an example output? I think no.
Consider removing them for other docs in this folder.